public/stylesheets/site/2.0/01-core.css

```diff
@@ -1,5 +1,6 @@
-/*==CORE:basic functional layout
-http://otwcode.github.com/docs/front_end_coding/css-shorthand*/
+/* == CORE: basic functional layout
+https://otwcode.github.io/docs/stylesheets-views.html#core
+*/
```

Comment:
For multiline comments, I decided to always put the `*/` on a separate line so we don't have a mix of same line and separate line based on whether or not there's a URL. However, I can't put the opening `/*` on a separate line because we have some methods that rely on certain text being the first line in our stylesheets, so we just sort of have to cope with it being weird.